### ISSUE TITLE
Error on Packing List page when trying to pack with no preferred foods

### ISSUE THREAD

**Issue by kenjiO** [freeCodeCamp/pantry-for-good#300]
On the Packing List page, if there is a customer with no foods listed in the **Preferred Foods** column and you check the checkbox to the left of the # column, then click the green **Mark selected customers as packed** button, I get an error: **Package validation failed: contents: Path `contents` is required.**.   If there are no foods in the preferred foods column the checkbox in the first column of the table should be disabled.

**Comment by jspaine** [comment]
This is done in #306, but the disabled checkbox state doesn't work well with the custom styled checkbox.

**Comment by kenjiO** [comment]
I'm still getting this problem
![screen shot 2017-11-22 at 3 39 48 pm](https://user-images.githubusercontent.com/5035181/33154210-b09be77c-cf9b-11e7-8228-23a390790f21.png)

**Comment by jcolborn-dropdeadgames** [comment]
I was reviewing this issue and I believe that this is only being seen in the seed data.  When I updated a customer and remove all of their Preferred Foods, I find that the checkbox is disabled.

**Comment by kristoferma** [comment]
I looked at this issue and it has been solved in the current version.

![screen shot 2018-04-30 at 9 09 37 am](https://user-images.githubusercontent.com/9106939/39429746-dd7078e0-4c59-11e8-90c9-27bd54084aad.png)

The checkbox is disabled if the preferred foods section is empty. There seems to be some kind of mixup with the CSS input/checkbox styles.

I will look into it and try to make the disabled checkbox look better
